Source/WebInspectorUI/UserInterface/Views/Variables.css

```diff
@@ -424,14 +424,10 @@ body {
         }
     }
 
-    &:is(.mac-platform.monterey, .mac-platform.big-sur):not(.docked) {
+    &:not(.docked) {
         --undocked-title-area-height: calc(27px / var(--zoom-factor));
     }
 
-    &:is(.mac-platform.catalina):not(.docked) {
-        --undocked-title-area-height: calc(22px / var(--zoom-factor));
-    }
-
     &.mac-platform {
```

Comment:
can we merge this with the above rule?

ditto anywhere else in this PR where we now have multiple declarations with the same selector

Reply:
Yes, I can do that. I avoided doing it in the first place to ensure the PR was easy to understand that only selectors were changed, not other styles. But I can do the merge now.